robot_skills/src/robot_skills/util/robot_constructor.py

```diff
@@ -1,5 +1,10 @@
-def robot_constructor(robot_name):
+import sys
+
+def robot_constructor(robot_name=None):
     """Construct a robot by it's name. Choices are amigo, sergio, mockbot"""
+    if not robot_name:
+        robot_name = sys.argv[1]
```

Comment:
+1 for this comment. It makes assumptions about the way this code is used and in general this seems not a sensible thing to do.